src/EditorFeatures/VisualBasicTest/CodeActions/InlineTemporary/InlineTemporaryTests.vb

```diff
@@ -4207,5 +4207,317 @@ End Class
+        ' comment
+        Dim a = 1, b = 2, 'ab
+            [||]c = 3 'c
+        Dim e = a
+        Dim f = b
+        Dim g = c
+    End Sub
+End Class
+"
+
+            Dim expected = "
+Class C
+    Sub M()
+        'c
+        ' comment
+        Dim a = 1, b = 2 'ab
+        Dim e = a
+        Dim f = b
+        Dim g = 3
+    End Sub
+End Class
+"
+
+            Await TestAsync(code, expected, compareTokens:=False)
+        End Function
+
+        <Fact(), Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)>
+        Public Async Function TestRetainComment7() As Task
+            Dim code = "
+Class C
+    Sub M()
+        ' comment
+        Dim a = 1, [||]b = 2, c = 3 'abc
+        Dim e = a
+        Dim f = b
+        Dim g = c
+    End Sub
+End Class
+"
+
+            Dim expected = "
+Class C
+    Sub M()
+        ' comment
+        Dim a = 1, c = 3 'abc
+        Dim e = a
+        Dim f = 2
+        Dim g = c
+    End Sub
+End Class
+"
+
+            Await TestAsync(code, expected, compareTokens:=False)
+        End Function
+
+        <Fact(), Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)>
+        Public Async Function TestRetainComment8() As Task
+            Dim code = "
+Class C
+    Sub M()
+        ' comment
+        Dim a = 1, b = 2, [||]c = 3 'abc
+        Dim e = a
+        Dim f = b
+        Dim g = c
+    End Sub
+End Class
+"
+
+            Dim expected = "
+Class C
+    Sub M()
+        ' comment
+        Dim a = 1, b = 2 'abc
+        Dim e = a
+        Dim f = b
+        Dim g = 3
+    End Sub
+End Class
+"
+
+            Await TestAsync(code, expected, compareTokens:=False)
+        End Function
+
+        <Fact(), Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)>
+        Public Async Function TestRetainComment9() As Task
+            Dim code = "
+Class C
+    Sub M()
+        ' comment
+        Dim a = 1, 'a
+            [||]b = 2,
+            c = 3 'c
+        Dim e = a
+        Dim f = b
+        Dim g = c
+    End Sub
+End Class
+"
+
+            Dim expected = "
+Class C
+    Sub M()
+        ' comment
+        Dim a = 1, 'a
+            c = 3 'c
+        Dim e = a
+        Dim f = 2
+        Dim g = c
+    End Sub
+End Class
+"
+
+            Await TestAsync(code, expected, compareTokens:=False)
+        End Function
+
+        <Fact(), Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)>
+        Public Async Function TestRetainComment10() As Task
+            Dim code = "
+Class C
+    Sub M()
+        ' comment
+        Dim a = 1, 'a
+            [||]b = 2, c = 3 'c
+        Dim e = a
+        Dim f = b
+        Dim g = c
+    End Sub
+End Class
+"
+
+            Dim expected = "
+Class C
+    Sub M()
+        ' comment
+        Dim a = 1, 'a
+            c = 3 'c
+        Dim e = a
+        Dim f = 2
+        Dim g = c
+    End Sub
+End Class
+"
+
+            Await TestAsync(code, expected, compareTokens:=False)
+        End Function
```

Comment:
do you have tests that inline the variable into multiple locations?  If so, we won't want to preserve the trivia in each case (likely we'll only want it for the first case).